### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by melih-unsal** [melih-unsal/DemoGPT#5] [Announcements]
### What's Changed

- Creates faster application
- Creates more robust application

<hr /><em>This discussion was created from the release <a href='https://github.com/melih-unsal/DemoGPT/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by AeonSunLVX** [comment]
Hey, great app! Question though — Where are the app files stored?

**Reply by melih-unsal** [reply]
Hi @AeonSunLVX , thanks for your attention.
Currently, the resulting file is stored in /tmp folder.
However, we will also add a folder selection to store these app files for the next update.

**Comment by melih-unsal** [comment]
Hi @AeonSunLVX , thanks for your contribution.
This kind of discussions are so valuable for us. If you are interested in this repo, you can support us by giving a ⭐
Thanks in advance 🙏🏻.

**Reply by melih-unsal** [reply]
Hey @AeonSunLVX , 

In the new release, there are 2 main changes:
- The code is displayed in the UI that you can directly copy
- The Windows-related bugs have been fixed.

Now, as we discussed before, the generated code is available on the UI.
The reason why we didn't add a folder selection is that only a single file is generated and it is easy to show on the UI.
However, for further releases, we will add a project folder path selection.

If you are interested in this repo, you can support us by giving us a star
Thanks in advance 🙏🏻.
